### PR TITLE
Merge test

### PR DESCRIPTION
**Pull request by thinktt**

### CODE REVIEW

File: routes/resources4.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    async.parallel([
+      getResourcePropertyTable, getDepartmentTable
+      ], function(err, results){
+          var collect = [];
+          for(var i = 0; i < departmentTable.length; i++){
+            var name = resourcePropertyTable.find(function(m){
+              return m['id'] == departmentTable[i]['directorId'];
+            });
+
+            if(name)
```
Comment (thinktt) [inline routes/resources4.js]: if an if statement is more than a short one liner always use brackets. If you ever go to the next line then use brackets.

File: routes/resources4.js [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+  getDirectors(req, res, next){
+    async.parallel([
+      getResourcePropertyTable, getDepartmentTable
+      ], function(err, results){
+          var collect = [];
+          for(var i = 0; i < departmentTable.length; i++){
+            var name = resourcePropertyTable.find(function(m){
+              return m['id'] == departmentTable[i]['directorId'];
+            });
+
```
Comment (thinktt) [inline routes/resources4.js]: If an if statement is ever more than a short one liner always use bracket @kaylamatteo  @chareth

File: routes/resources4.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+      } // end of if  resourcePropertyTable[i]['departmentId']
+      else continue;
+
+
+    if(resourcePropertyTable[i]['typeOptionId']){
+      var type = resourceOptionTable.find(function(m){
+        return m['id'] == resourcePropertyTable[i]['typeOptionId'];
+      });
+        resourcePropertyTable[i]['type'] = type['value'];
+      } //end if(resourcePropertyTable[i]['typeOptionId'])
```
Comment (thinktt) [inline routes/resources4.js]: @chareth @kaylamatteo 
avoid bracket notation unless there is a good reason. Use dot notation whenever you can.
 resourcePropertyTable[i]['type'] = type['value']; 
should become 
resourcePropertyTAble[i].type = type.value

File: public/views/admin.html [hunk 4]
```diff
@@ -9,9 +9,41 @@ <h2 class="text-center">
 </h2>
 <hr class="clear">
 <div class="panel-body">
+
 <!-- Build Report Link-->
 <div class="col-md-6">
     <div class="panel panel-default">
+        <div class="bg-info add-padding text-center" style="max-height: 70px;min-height: 70px">
```
Comment (ashpc) [inline public/views/admin.html]: You can use 'ReportController' as the controller for this panel and use the monthsList to render the dropdown using ng-repeat.

File: public/views/reports/unallocated.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+                            <td>{{resource.manager}}</td>
+                            <td>{{resource.org}}</td>
+                            <td class="text-center" ng-show="isAdmin() ||  isMyResource(resource.manager) || isProjectManager()">
+                                <a href="/forecasts/all/{{resource.id}}" style="margin-right: 10px">
+                                    <i class="fa fa-money text-success"></i>
+                                </a>
+                                <a href="/resources/{{resource.id}}?edit=true" style="margin-right: 10px">
+                                    <i class="fa fa-pencil-square-o"></i>
+                                </a>
+                                <i class="fa fa-times text-danger" ng-click="deleteResourceConfirm(resource)"></i>
```
Comment (ashpc) [inline public/views/reports/unallocated.html]: I don't think you will need resourceDeletion from this report page as it belongs to the resource page.So go ahead and remove this.

**Pull request merged**